server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/RemoteDelivery.java

```diff
@@ -114,6 +114,15 @@
  * the ability to perform their own problem resolutions.</li>
  * <li><b>debug</b> (optional) - a Boolean (true/false) indicating whether debugging is on. Default is false.</li>
  * </ul>
+ * <br/>
+ * <b>Security:</b><br/>
+ * You can use the <i>mail.smtp.ssl.enable</i> javax property describe above to force SMTP outgoing delivery to default to SSL
```

Comment:
we should recommend using tls.enable instead, no?

Reply:
What do you mean by "TLS"? How do you achieve this using javax.mail.smtp? What precise property would you advice users to use?

Reply:
sorry, I meant using `<startTLS>true</startTLS>` in RemoteDelivery mailet configuration.
(and yes we need to properly document what can be configured by the RemoteDelivery mailet, what can be configured by mail.smtp properties, and how it interracts between each other).

Reply:
s/describe/described